autonomy/cli/deploy.py

```diff
@@ -162,6 +162,8 @@ def deploy_group(
     help="Use local tendermint chain setup.",
 )
 @click.option("--image-version", type=str, help="Define runtime image version.")
+@click.option("--agent-cpu", type=float, help="Set agent CPU usage limit.")
+@click.option("--agent-memory", type=int, help="Set agent memory usage limit.")
```

Comment:
See my comments elsewhere. These are limits and we should be explicit about the name imo. Because there are also "requests" which imo should be configurable too.